test/test_network_constructors.jl

```diff
@@ -162,6 +165,46 @@ end
     end
 end
 
+@testset "Test Locational Marginal Prices between DC lossless with PowerModels vs StandardPTDFModel" begin
+    network = [DCPPowerModel, StandardPTDFModel]
+    sys = build_system("c_sys5")
+    dual_constraint = [
+        [PSI.make_constraint_name(PSI.NODAL_BALANCE_ACTIVE, PSY.Bus)],
+        [:RateLimit_lb__Line, :RateLimit_ub__Line, :nodal_balance, :network_flow],
+    ]
+
+    parameters = [true, false]
+    ptdf = build_PTDF5()
+    lmps = []
+    for (ix, net) in enumerate(network), p in parameters
+        ps_model = OperationsProblem(
+            TestOpProblem,
+            net,
+            sys;
+            optimizer = OSQP_optimizer,
+            use_parameters = p,
+            PTDF = ptdf,
+            constraint_duals = dual_constraint[ix],
+        )
+        construct_device!(ps_model, :Thermal, thermal_model)
```

Comment:
Why is this test calling `construct_device` here? Please use a template in the test.

Reply:
this is similar to what you have in other tests in that file, so I just followed that

Reply:
Some of the tests are old before the implementation of Templates, we need to refactor the testing to remove the individual construct device pattern